lang/c_sharp/README.md

```diff
@@ -0,0 +1,23 @@
+# Notes for patching
+
+The origin `.Net` runtime creates debug pipes using `SYS_mknodat` syscall, which is not yet implemented in Unikraft.
+
+Setting env `COMPlus_EnableDiagnostics=0` or `DOTNET_EnableDiagnostics=0` could prevent this.
+
+For convince, we patched [EXTERNAL_EnableDiagnostics](https://github.com/dotnet/runtime/blob/v7.0.7/src/coreclr/inc/clrconfigvalues.h#L162)
+to be `0` by default.
+
+And it only affects `/usr/share/dotnet/shared/Microsoft.NETCore.App/7.0.7/libcoreclr.so`.
+
+# Notes for procfs
+
+The `.Net` runtime uses [__pthread_getattr_np](https://github.com/bminor/glibc/blob/glibc-2.37/nptl/pthread_getattr_np.c#L85)
+in [CPalThread::GetStackBase()](https://github.com/dotnet/runtime/blob/v7.0.7/src/coreclr/pal/src/thread/thread.cpp#L2683)
+and [CPalThread::GetStackLimit()](https://github.com/dotnet/runtime/blob/v7.0.7/src/coreclr/pal/src/thread/thread.cpp#L2723).
+
+The `glibc` implementation reads `/proc/self/maps` to get stack addresses, see the condition [here](https://github.com/bminor/glibc/blob/glibc-2.37/nptl/pthread_getattr_np.c#L129).
+
+You have to change the content of `/proc/1/maps` in your `rootfs`, to make your range of stack address contains the real
+stack area. Otherwise, the runtime will crash with `Failed to create CoreCLR, HRESULT: 0x8007000E`.
```

Comment:
```suggestion
You have to change the content of `/proc/1/maps` in your `rootfs`, to make your range of stack address contains the real stack area.
Otherwise, the runtime will crash with `Failed to create CoreCLR, HRESULT: 0x8007000E`.
```